### PR TITLE
Upgrade operator sdk version

### PR DESCRIPTION
**Pull request by camilamacedo86**
## Motivation
Release of a new version of operator-SDK

## What
Upgrade the version of the framework

## Why
- Keep it easy to maintain and to be update

## Verification Steps
1. Check the CI
2. Install it locally and ensure that all was done with success by running `make install`
IMAGE: docker.io/cmacedo/mobile-security-service-operator:upgrade
## Checklist:

- [X] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task
- [ ] TODO

## Motivation
Release of a new version of operator-SDK

## What
Upgrade the version of the framework

## Why
- Keep it easy to maintain and to be update

## Verification Steps
1. Check the CI
2. Install it locally and ensure that all was done with success by running `make install`
IMAGE: docker.io/cmacedo/mobile-security-service-operator:upgrade
## Checklist:

- [X] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task
- [ ] TODO

## Aditional Notes:

```shell
 $ oc get all
NAME                                                    READY     STATUS    RESTARTS   AGE
pod/mobile-security-service-bcbd4d49f-k48dj             1/2       Running   2          45s
pod/mobile-security-service-db-56d7cb55b-v8cxd          1/1       Running   0          17s
pod/mobile-security-service-operator-689dd57855-m8wm7   1/1       Running   0          1m

NAME                                               TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)             AGE
service/mobile-security-service-application        ClusterIP   172.30.4.110     <none>        3000/TCP            45s
service/mobile-security-service-db                 ClusterIP   172.30.60.249    <none>        5432/TCP            17s
service/mobile-security-service-operator-metrics   ClusterIP   172.30.105.246   <none>        8383/TCP,8686/TCP   47s
service/mobile-security-service-proxy              ClusterIP   172.30.182.214   <none>        80/TCP              45s

NAME                                               DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/mobile-security-service            1         1         1            0           47s
deployment.apps/mobile-security-service-db         1         1         1            1           17s
deployment.apps/mobile-security-service-operator   1         1         1            1           1m

NAME                                                          DESIRED   CURRENT   READY     AGE
replicaset.apps/mobile-security-service-bcbd4d49f             1         1         0         46s
replicaset.apps/mobile-security-service-db-56d7cb55b          1         1         1         17s
replicaset.apps/mobile-security-service-operator-689dd57855   1         1         1         1m

NAME                             HOST/PORT                                            PATH      SERVICES                        PORT      TERMINATION   WILDCARD
route.route.openshift.io/route   route-mobile-security-service.192.168.64.30.nip.io             mobile-security-service-proxy   web       edge          None
```

Screenshot 2019-08-12 at 16 03 51

### CODE REVIEW

**Comment by camilamacedo86** [comment]
Hi @wei-lee this PR will also solve the issue; https://issues.jboss.org/browse/AEROGEAR-9736
Could we move forward with and do a new release? ( PS. the new release will be required anyway)

File: README.adoc [hunk 1]
```diff
@@ -126,6 +126,36 @@ Status:
 
 NOTE: To troubleshooting the installation if it does not complete successfully check the status of its CRs. Note that all objects which are managed by them has an respective status field on it. E.g `Deployment Status`
 
+Following all an expected result from the command `oc get all` executed in the namespace where the operator was installed after all required resources managed by it be crated with success.
```
Comment (laurafitzgerald) [inline README.adoc]: crated -> created

**Comment by laurafitzgerald** [comment]
@camilamacedo86 I just noticed this one is a little older. Is this required for the current release?

**Comment by camilamacedo86** [comment]
Hi @laurafitzgerald, 

Thank you for the review. It is not required for the delivery of RHMI.

**Comment by camilamacedo86** [comment]
I am closing this one because we will not do it now and it is outdated.

**Pull request closed without merging**